src/backend/src/services/design-reviews.services.ts

```diff
@@ -0,0 +1,179 @@
+  }
+
+  /**
+   * Deletes a design review
+   * @param submitter the user who deleted the design review
+   * @param designReviewId the id of the design review to be deleted
+   */
+
+  static async deleteDesignReview(submitter: User, designReviewId: string): Promise<DesignReview> {
+    const designReview = await prisma.design_Review.findUnique({
+      where: { designReviewId }
+    });
+
+    if (!designReview) throw new NotFoundException('Design Review', designReviewId);
+
+    if (!(isAdmin(submitter.role) || submitter.userId === designReview.userCreatedId))
+      throw new AccessDeniedAdminOnlyException('delete design reviews');
+
+    if (designReview.dateDeleted) throw new DeletedException('Design Review', designReviewId);
+
+    const deletedDesignReview = await prisma.design_Review.update({
+      where: { designReviewId },
+      data: { dateDeleted: new Date(), userDeleted: { connect: { userId: submitter.userId } } },
+      ...designReviewQueryArgs
+    });
+
+    return designReviewTransformer(deletedDesignReview);
+  }
+
+  /**
+   * Retrieves a single design review
+   *
+   * @param submitter the user who is trying to retrieve the design review
+   * @param designReviewId the id of the design review to retrieve
+   * @returns the design review
+   */
+  static async getSingleDesignReview(submitter: User, designReviewId: string): Promise<DesignReview> {
+    const designReview = await prisma.design_Review.findUnique({
+      where: { designReviewId },
+      ...designReviewQueryArgs
+    });
+
+    if (!designReview) throw new NotFoundException('Design Review', designReviewId);
+
+    if (designReview.dateDeleted) throw new DeletedException('Design Review', designReviewId);
+
+    return designReviewTransformer(designReview);
+  }
+
+  /**
+   * Edits a Design_Review in the database
+   * @param user the user editing the design review (must be leadership)
+   * @param designReviewId the id of the design review to edit
+   * @param dateScheduled the date of the design review
+   * @param teamTypeId the team that the design_review is for (software, electrical, etc.)
+   * @param requiredMembers required members for the design review
+   * @param optionalMembers optional members for the design review
+   * @param isOnline is the design review online (IF TRUE: zoom link should be requried))
+   * @param isInPerson is the design review in person (IF TRUE: location should be required)
+   * @param zoomLink the zoom link for the design review meeting
+   * @param location the location for the design review meeting
+   * @param docTemplateLink the document template link for the design review
+   * @param status see Design_Review_Status enum
+   * @param attendees the attendees for the design review (should they have any relation to the other shit / can't edit this after STATUS: DONE)
+   * @param meetingTimes meeting time must be between 0-84 (Monday 12am - Sunday 12am, 1hr minute increments)
```

Comment:
this should say 83/ the number should be 83 everywhere (84 boxes, indexed starting at 0, so the last one is 83)